kickstart/disklayout_atomic.erb

```diff
@@ -0,0 +1,14 @@
+<%#
+kind: ptable
+name: Kickstart Atomic
+oses:
+- CentOS-Atomic 7
```

Comment:
These OSs can't run Puppet for the moment, so they're kind of irrelevant for the moment. Since the names cannot contain spaces I chose to hyphenate Atomic.